qa/TL1_custom_src_pattern_build/test.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash -e
 
 build_and_check() {
+  pip install black dm-tree
```

Comment:
Why do we need that?

Reply:
OK, we install this in the base test container.

Reply:
I mean, I thought our `setup.py` already requires dm-tree. Isn't that relevent here?

Reply:
With stub generation you import DALI during the build without installing it, so your build env needs to have DALI dependencies beforehand. So `make -j` needs it already.

Reply:
Removed.

Reply:
I see, thanks!